xonsh/foreign_shells.py

```diff
@@ -11,12 +10,12 @@
 import warnings
 import functools
 import collections.abc as abc
+from collections import Sequence
```

Comment:
Please remove this line, this is a non-lazy import.  Sequence should be refered to via `abc.Sequence`